sklearn/cross_validation.py

```diff
@@ -632,11 +638,13 @@ class ShuffleSplit(BaseShuffleSplit):
     n_iter : int (default 10)
         Number of re-shuffling & splitting iterations.
 
-    test_size : float (default 0.1), int, or None
+    test_size : float (default 0.1), int, or None,
```

Comment:
On 05/04/2015 11:37 PM, SnakeHunt2012 wrote:
>
> Finally, I think you are right, it will make me puzzle if the 
> docstring dosen't match the parameter list. It's enough to explain it 
> afterwards.
>
Yeah that was my thinking, too.